### DISCUSSION TITLE
502 Bad Gateway Docker

### DISCUSSION THREAD

**Discussion by danielmochalov27** [NginxProxyManager/nginx-proxy-manager#2067] [Q&A]
Hello everyone,
I have npm installed and other docker applications on the same network. When I do a ssh tunnel to my dedicated machine I can access the systems directly no problem. When I try to run it though npm I get 502 Bad Gateway and my docker container I’m trying to run through npm says, "GET / HTTP/1.1" 302 562 "-" "curl/7.64.0". I have some images attached showing what I’m trying to do. I tried bunch of fixes searching in google and GitHub nothing I tried worked.

Thank you for any help

<img width="498" alt="E36D4D1F-92C0-478A-8DA9-88C5017F7E34" src="https://user-images.githubusercontent.com/40433547/169935268-6011641a-c7b6-437b-91e2-b7e1f809aa50.png">
<img width="469" alt="129355E3-7EDE-4A77-9EAB-14FDA702B8B8" src="https://user-images.githubusercontent.com/40433547/169935271-5192e12e-a3a8-4dbe-99d1-7e399aa8868f.png">

**Comment by Fazatholomew** [comment]
Having the same problem. Logs said: "failed (111: Connection refused) while connecting to upstream"
